src/harness/compilerRunner.ts

```diff
@@ -253,10 +250,10 @@ class CompilerBaselineRunner extends RunnerBase {
                     // These types are equivalent, but depend on what order the compiler observed
                     // certain parts of the program.
 
-                    const allFiles = toBeCompiled.concat(otherFiles).filter(file => !!program.getSourceFile(file.unitName));
+                    const allFiles = toBeCompiled.concat(otherFiles).filter(file => !!result.program.getSourceFile(file.unitName));

```

Comment:
Since you use `result.program` several times, just extract it out to a local constant named `program`.